elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java

```diff
@@ -472,7 +472,7 @@ public static BlockReason getByNumber(String blockReason) {
 	public static final Boolean DATA_BASE_REGISTER_DEFAULT_FLAG = false;
 	public static final Integer DATA_BASE_ADMIN_ID = 0;
 	public static final String DATA_BASE_ADMIN_LOGIN = "admin";
-	public static final String DATA_BASE_ADMIN_PASSWORD = "f7e3c24e1a04758097f69be41aa3cf18";
```

Comment:
Please remove it as it's a local change. 🗑️

Reply:
I did it.